db/changes.go

```diff
@@ -171,6 +172,151 @@ func (db *Database) AddDocInstanceToChangeEntry(entry *ChangeEntry, doc *Documen
 	}
 }
 
+func (db *Database) buildRevokedFeed(singleChannelCache SingleChannelCache, options ChangesOptions, triggeredBy uint64) <-chan *ChangeEntry {
+	feed := make(chan *ChangeEntry, 1)
+	sinceVal := options.Since.Seq
+
+	queryLimit := db.Options.CacheOptions.ChannelQueryLimit
+	requestLimit := options.Limit
+
+	paginationOptions := options
+	paginationOptions.Since.Seq = 0
```

Comment:
In the case of an interrupted and resumed revocation (i.e. since value like seq:revocationSeq), we don't want to start again from zero.  The same is handled for backfill on the normal changesFeed today using the since value in ChangesOptions (I think), it feels like we want to do the same here.

Reply:
Added handling in the latest commit for this. Should handle this scenario and allow a resume if possible.